lib/cretonne/src/dominator_tree.rs

```diff
@@ -323,6 +329,72 @@ impl DominatorTree {
     }
 }
 
+impl DominatorTree {
+    /// When splitting an `Ebb` using `Layout::split_ebb`, you can use this method to update
+    /// the dominator tree locally rather than recomputing it.
+    ///
+    /// `old_ebb` is the `Ebb` before splitting, and `new_ebb` is the `Ebb` which now contains
+    /// the second half of `old_ebb`.
+    pub fn recompute_split_ebb(&mut self, old_ebb: Ebb, new_ebb: Ebb, split_jump_inst: Inst) {
```

Comment:
But actually in order to do the `debug_assert()` I would need to add the `dfg` and `layout` as arguments to the function. Is it worth it since they're not going to be used in release mode?